pagopa/io-app
# feat(IDPay): [IODPAY-206] IDPay Onboarding playground redesign

## Short description
This PR adds a redesign of the IDPay Onboarding Playground, which simplifies the selection of test initiatives available thorugh the [io-dev-server](https://github.com/pagopa/io-dev-api-server).

https://user-images.githubusercontent.com/6160324/229274899-239db961-1a09-4711-afde-ca65263e4183.mp4


Note: the ❌ and ✅ next to the services are for the expected onboarding result.

## List of changes proposed in this pull request
- [.env.local](https://github.com/pagopa/io-app/pull/4506/files#diff-74dd9d08974aed4457940214a64b43ce175d062d74e72309b003f02c24f98b30):
  - Added local server url
- [ts/screens/profile/playgrounds/IDPayOnboardingPlayground.tsx](https://github.com/pagopa/io-app/pull/4506/files#diff-c512cd692a35cd0367dea9f41b1366adf200d142955bcbfbd4398deea3030a93)
  - Added a list of pressable testing initiatives

## How to test
- Start the local dev server on your machine.
- Make sure you have enabled the IDPay feature in the app through the profile screen.
- From the profile screen, open the IDPay Onboarding Playground
